src/ua.c

```diff
@@ -983,6 +983,7 @@ int ua_alloc(struct ua **uap, const char *aor)
 	if (err)
 		goto out;
 
+	ua_set_media_af(ua, ua->acc->maf);
```

Comment:
now we have duplicated info in both ua and account about `mediaaf`.

perhaps consider to remove it from ua and store it only in `struct account` ?

Reply:
Right, removing from ua seems to be the right solution. Although it changes the API.